src/main.c

```diff
@@ -506,24 +624,39 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
     PIPV4_HEADER Ipv4Hdr;
     PIPV6_HEADER Ipv6Hdr;
 
-    if (!IsEqualGUID(&ev->EventHeader.ProviderId, &NdisCapId) ||
+    if ((!IsEqualGUID(&ev->EventHeader.ProviderId, &NdisCapId) && !IsEqualGUID(&ev->EventHeader.ProviderId, &RasNdisWanCapId)) ||
```

Comment:
This check isn't quite right, because it won't filter out e.g. ndiscap events that coincidentally have the same event ID as tidRRasNdisWanSendPckts (even if such events don't exist today, they could exist tomorrow when somebody adds them).